specification.markdown

```diff
@@ -404,23 +404,24 @@ PathMaxOccurrence     = 1*DIGIT ; no upper limit
-
-The GlobalParentOccurence part is interpreted as an ABNF Variable Repetition. The repetition amounts correspond to the amount of unspecified Parent Element levels there can be between the EBMLFixedParent and the actual EBMLElementPath.
-
-PathMinOccurrence represents the minimum number of element path required between the EBMLFixedParent and the Global Element EBMLElementPath. For example 0 means the EBMLElementPath can be right after the EBMLFixedParent, 1 means there has to be at least an element between the EBMLFixedParent and the EBMLElementPath. If PathMinOccurrence is not present then that EBML Element has an PathMinOccurrence value of 0.
+The `@path` value MUST be unique within the EBML Schema. The `@id` value corresponding to this `@path` MUST NOT be defined for use within another EBML Element with the same EBMLParentPath as this `@path`.
 
-PathMaxOccurrence represents the maximum number of element path possible between the EBMLFixedParent and the Global Element EBMLElementPath. It cannot have the value 0 as it would be the Global Element can only be found right after the EBMLFixedParent, in which case it's not a Global Element anymore. If PathMaxOccurrence is not present then there is no upper bound for the permitted number of occurrences of element path possible between the EBMLFixedParent and the Global Element EBMLElementPath.
+In some cases the path contains an GlobalPlaceholder.
+A path with a GlobalPlaceholder defines a Global Element; see (#global-elements).
+If GlobalPlaceholder is not present then the Element is not a Global Element and only has one fixed EBMLParentPath.
+Any path that starts with the EBMLParentPath of the Global Element and matches the occurrences found in the GlobalParentOccurence is a valid path for the Global Element.
 
-If the path contains an EBMLPathAtomRecursive part, the EBML Element can occur within itself recursively (see (#recursive)).
+The GlobalParentOccurence part is interpreted as an ABNF Variable Repetition.
+The repetition amounts correspond to the amount of unspecified valid Elements that can be found in place of the GlobalPlaceholder for the path to be valid.
 
-As an example, a `path` of `1*(\Segment\Info)` means the element Info is found inside the Segment elements at least once and with no maximum iteration. An element SeekHead with path `0*2(\Segment\SeekHead)` may not be found at all in its Segment parent, once or twice but no more than that.
+PathMinOccurrence represents the minimum number of element path required between the EBMLParentPath and the EBMLLastPath. For example 0 means the EBMLLastPath can be right after the EBMLParentPath, 1 means there has to be at least an element between the EBMLParentPath and the EBMLLastPath. If PathMinOccurrence is not present then that GlobalPlaceholder has a PathMinOccurrence value of 0.
```

Comment:
`the minimum number of element path` could be better worded. It feels unclear to me. 

Actually the first sentence is hard to process. For instance with `/Root/(2*)/global` if PathMinOccurrence=2 and there needs to be at least 2 `number of element path required between the EBMLParentPath and the EBMLLastPath`. Does that mean?

```
/Root
/Root/ChildA
/Root/ChildB
/Root/global
```

or
```
/Root/ChildA/ChildB/global
```

Under different interpretations ChildA and ChildB are `between` Root and global.

Reply:
That's `\Root\ChildA\ChildB\global`. I'll see if I can find something bettter.

Reply:
Added some more detailed example, hopefully not too verbose.